debian/control

```diff
@@ -39,8 +39,10 @@ Depends: eos-shell-content (= ${binary:Version}),
          ${misc:Depends},
          ${perl:Depends},
          debhelper,
+         libappstream-glib,
```

Comment:
Sorry, I always forget the *8* suffix that we have for some reason on Debian. On Fedora it's just *libappstream-glib*.
Packaging is so much fun!